client/post-editor/media-modal/gallery/edit-item.jsx

```diff
@@ -33,7 +33,7 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { site, item } = this.props;
+		const { site, item, showRemoveButton = true } = this.props;
```

Comment:
Should we set a default prop for `showRemoveButton` instead of doing so inline here?